docs/about/about.md

```diff
@@ -1,16 +1,16 @@
-# About ODS
+# About TODS
 
-The Operational Data Standard ("ODS") is managed by [MobilityData](https://mobilitydata.org) serving as the [ODS Manager](../governance/governance.md#ods-manager) under the authority of the [ODS Board](../governance/governance.md#ods-board-of-directors) with basic monetary support through 2025 from the [California Integrated Travel Project (Cal-ITP)](https://cal-itp.org).
+The Transit Operational Data Standard ("TODS") is managed by [MobilityData](https://mobilitydata.org) serving as the [TODS Manager](../governance/governance.md#TODS-manager) under the authority of the [TODS Board](../governance/governance.md#TODS-board-of-directors) with basic monetary support through 2025 from the [California Integrated Travel Project (Cal-ITP)](https://cal-itp.org).
 
-ODS was [initially developed](spec-development.md) in 2021-2022 by the [ODS Working Group](contributors.md) (now known as *contributors*) using resources from the [California Integrated Travel Project (Cal-ITP)](https://cal-itp.org) in order to:
+TODS was [initially developed](spec-development.md) in 2021-2022 by the [TODS Working Group](contributors.md) (now known as *contributors*) using resources from the [California Integrated Travel Project (Cal-ITP)](https://cal-itp.org) in order to:
 
 1. Create a common standard for transit operational data
 2. Improve efficiency in dispatching, planning, scheduling, driver shift management, and other back-end transit operations
 3. Reduce the burden of generating data for reporting (FTA and other)
 
-As of 2023:
+As of 2024:
 
-* ODS v1.0 is leveraged by several software vendors and transit agencies,
-* ODS v2.0 is under active development,
+* TODS v1.0 is leveraged by several software vendors and transit agencies,
+* TODS v2.0 is under active development,
```

Comment:
TODS v2.0 has been adopted and is ready for implementation

Reply:
fixed in [9e9ccce](https://github.com/cal-itp/operational-data-standard/pull/78/commits/9e9ccce13f58e8e5e11557012c69c907cd27d2ce)